src/Canvas/Pixel.elm

```diff
@@ -0,0 +1,227 @@
+module Canvas.Pixel
+    exposing
+        ( put
+        , putMany
+        , line
+        , rectangle
+        , bezier
+        )
+
+import Canvas exposing (Canvas, Size, Point, DrawOp(..))
+import Color exposing (Color)
+import Array exposing (Array)
+
+
+put : Color -> Point -> DrawOp
```

Comment:
I don't know that I'm a fan of `put`, `putMany`, and `putHelp`. Setting individual pixels like this isn't as performant as grabbing a region, being able to modify the whole region, then blitting that whole region back onto the image, which is generally how I would expect the `getImageData` and `putImageData` methods to work. Let's keep these in for now, though -  there is probably value in setting a single pixel, but we could definitely boost the performance for the more advanced drawing methods in this module.